tnt/src/main/java/org/dcsa/conformance/standards/tnt/party/TntPublisher.java

```diff
@@ -112,13 +130,167 @@ private void supplyScenarioParameters(JsonNode actionPrompt) {
   public ConformanceResponse handleRequest(ConformanceRequest request) {
     log.info("TntPublisher.handleRequest(%s)".formatted(request));
 
-    JsonNode jsonResponseBody =
-        JsonToolkit.templateFileToJsonNode(
-            "/standards/tnt/messages/tnt-220-response.json", Map.ofEntries());
+    Map<String, List<AttributeMapping>> attributeMappings = AttributeMapping.initializeAttributeMappings();
+
+    ObjectMapper mapper = new ObjectMapper();
```

Comment:
Please use the existing `OBJECT_MAPPER` object, which you can import. Creating the ObjectMapper every time is taking resources.

Reply:
done